<launch>


<!--DÃ©monstrateur: port 50011-->
<!--Rocket (grand robot); port 50010-->

  <!-- robot_ip: IP-address of the robot's socket-messaging server -->
  <arg name="robot_ip" />
  <arg name="port" />
  <arg name="cables" default="8"/>
    
  <!-- The max_velocity parameter is only used for debugging in the ur_driver. It's not related to actual velocity limits -->
  <arg name="max_velocity" default="10.0"/> <!-- [rad/s] -->
    <param name="number_of_cables" type="int" value="$(arg cables)"/>

<!--  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" /> -->

  <!-- driver -->
  <node name="ros_driver" pkg="br_driver" type="ros_driver" output="screen" >

  <!-- copy the specified IP address to be consistant with ROS-Industrial spec. -->
    <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />
    <param name="commuinication_port" type="int" value="$(arg port)" />

  </node>


	<rosparam command="load" file="$(find br_driver)/config/robot_parameters.yaml" />


</launch>
